CHARTER.adoc

```diff
@@ -0,0 +1,195 @@
+
+The objectives of GeoZarr conventions includes:
+
+1. Compatibility: Ensuring easy compatibility with popular mapping and data analysis tools such as GDAL, Xarray, ArcGIS, and QGIS, enabling seamless integration into existing workflows.
+2. Visualisation: Simplifying the creation and display of geospatial data in web browsers without the need for complex workarounds, making geospatial information more accessible to users.
+3. Dimensions: Supporting multidimensional data, such as hyperspectral and altitude information, to address diverse geospatial data requirements.
+4. Data Discovery: Providing metadata for discovering, accessing, and retrieving the data, including subsets or different arrangements of the data.
+5. Mixing Data: Facilitating the combination of different types of geospatial data, including satellite images, elevation maps, and weather models, to create comprehensive and informative datasets.
+8. Flexibilty: Allowing scientists and researchers to work with diverse data types and projections in their preferred software and programming languages, promoting flexibility and adaptability in geospatial data processing and analysis.
+
+Specifically, the convention should provide guidance to standardize the approach for encoding various aspects of geospatial data, including:
+
+* Multiple related variables with heterogeneous coordinates (e.g., children or linked datasets)
+* Multiple resolutions of the data, possibly leveraging multiscale drafts
+* Data subsets that are only available at certain resolutions
+* Multiple projections
+* Multi-dimensional optimizations (rechunking)
+* Supporting typical Earth observation (EO) products (for example, how to encode multispectral bands)
+* Encoding of time series that have not been normalized
+* Accessing the symbology of the corresponding data
+
+=== Statement of relationship of planned work to the current OGC standards baseline
+As the existing draft GeoZarr metadata utilizes Climate and Forecast (CF) attributes, there is an expected relationship with the OGC CF-netCDF Data Model Extension Standard. If necessary, any adjustments or enhancements required for the GeoZarr specification will be considered as a proposal to improve CF conventions as well.
+
+
+=== What is Out of Scope?
+In early conversations around creating a draft GeoZarr specification, concerns arose multiple times around the CF encoding of CRS which may pose issues, see https://github.com/zarr-developers/geozarr-spec/issues/20. While these concerns will be discussed and suggestions created for potentially updating CF conventions, if resolutions cannot be made with the GeoZarr specification, we consider out of scope waiting on any subsequent updates to CF convetions to reflect these suggestions. 
```

Comment:
This seems counter to the sentence (and discussion) above in the Purpose section:
> The GeoZarr SWG will also work on improving the Climate and Forecast (CF) conventions if necessary, particularly for alternative coordinate reference system (CRS) encoding if relevant.

I agree GeoZarr shouldn't wait forever for changes to CF. However, working with the CF community for some reasonable amount of time certainly seems within scope for GeoZarr.

Perhaps this should be removed and some mention of not being totally blocked by CF disagreements should be added in the Purpose section or the Relationshop with Other OGC Standards section?

Not sure what should be out of scope. Perhaps something generic and simple like:
> The GeoZarr SWG will only consider matters related to encoding geospatial data in Zarr.